backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java

```diff
@@ -3033,6 +3033,13 @@ void writeVideo(VmDevice device) {
         writer.writeEndElement();
     }
 
+    private String getVideoType(String deviceType) {
+        if (!deviceType.equals("vga")) {
+            return deviceType;
+        }
+        return vmInfoBuildUtils.isVirtioVgaUsed(vm.getVmOsId(), vm.getCompatibilityVersion()) ? "virtio" : "vga";
```

Comment:
unless we change devices monitoring to cope with this change, the type of the device would change in the database and that could affect the UI/API, no?

Reply:
The video device in the DB is created with device type 'vga' and correlated by device ID. The devices monitoring do not replace device type when updating existing devices from VDSM data. So the video device will remain to be 'vga'.

Reply:
ack, thanks for checking, sounds good then